behaviour/graql/language/delete.feature

```diff
@@ -427,33 +596,70 @@ Feature: Graql Delete Query
       | FR    | ALEX | BOB  |
 
 
-  # this scenario should be identical in behaviour to the above, only the match differs
+  Scenario: when deleting duplicate role players in multiple statements, it still removes the total number you asked to delete
```

Comment:
good but i dont think we need words like "still", they are not very "objective"?

Reply:
Ok clearing this one out